pasystem/pasystem-impl/impl/src/resources/db/migration/mysql.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE IF NOT EXISTS `pasystem_popup_screens` (
+  `uuid` VARCHAR(255) PRIMARY KEY,
```

Comment:
I think these UUID database fields should be 36 fixed chars instead of varchar(255)